packages/integration-test/test/scheduled/erc777-stream.test.ts

```diff
@@ -59,6 +59,16 @@ describe('ERC777 SuperFluid detection test-suite', () => {
       tokenAddress: '0x745861aed1eee363b4aaa5f1994be40b1e05ff90', // FAU
     });
 
+    // When added, there are 2 mockRequest variables + find ERC777 token
```

Comment:
you don't need to support erc777 in this PR

Reply:
Should I delete the test then?